src/components/CSSTable.tsx

```diff
@@ -0,0 +1,268 @@
+import { debounce, List, ListItem, Stack } from '@patternfly/react-core'
+import { Table, Thead, Th, Tr, Tbody, Td } from '@patternfly/react-table'
+import { useState } from 'react'
+import LevelUpAltIcon from '@patternfly/react-icons/dist/esm/icons/level-up-alt-icon'
+import * as tokensModule from '@patternfly/react-tokens/dist/esm/componentIndex'
+import React from 'react'
+import { CSSSearch } from './CSSSearch'
+import { AutoLinkHeader } from './AutoLinkHeader'
+
+type Value = {
+  name: string
+  value: string
+  values?: string[]
+}
+
+type FileList = {
+  [key: string]: {
+    name: string
+    value: string
+    values?: Value[]
+  }
+}
+
+type List = {
+  selector: string
+  property: string
+  token: string
+  value: string
+  values?: string[]
+}
+
+type FilteredRows = {
+  cells: React.ReactNode[]
+  isOpen?: boolean
+  details?: { parent: number; fullWidth: boolean; data: React.ReactNode }
+}
+
+interface CSSTableProps extends React.HTMLProps<HTMLDivElement> {
+  cssPrefix: string
+  hideSelectorColumn?: boolean
+  selector?: string
+  debounceLength?: number
+  autoLinkHeader?: boolean
+}
+
+const isColorRegex = /^(#|rgb)/
+const mappingAsList = (property: string, values: string[]) => (
+  <List isPlain>
+    <ListItem>{property}</ListItem>
+    {values.map((entry: string) => (
+      <ListItem key={entry} icon={<LevelUpAltIcon className="rotate-90-deg" />}>
```

Comment:
It looks like the rotate isn't working? Though maybe something is just funky on my end: 
<img width="505" alt="image" src="https://github.com/user-attachments/assets/fc7a41fb-41d5-41f1-b121-cb0ba95d388b" />

Reply:
Thanks, accidentally dropped this class!